### PR TITLE
Fix tests

### LINKED ISSUE #563: Look into 11 skipped front-end tests
For `npm test`. Hit `a` to run them all.
`Test Suites: 3 skipped, 7 passed, 7 of 10 total
Tests:       11 skipped, 18 passed, 29 total`

### PR DESCRIPTION
**Pull request by yunnie**
### What issue is this solving?
Please connect this Pull Request to the issue it is resolving by using the controls on the right side of this PR.

Closes #563

### Any helpful knowledge/context for the reviewer?
Is a `npm install` necessary? No
Any special requirements to test? No

### Please make sure you've attempted to meet the following coding standards
- [ x] Code builds successfully
- [ x] Code has been tested and does not produce errors
- [ x] Code is readable and formatted
- [ x] There isn't any unnecessary commented-out code
- [ x] There aren't any unnecessary commits or files changed (shouldn't touch the /stashed folder unless the issue requests it)

If you're having trouble meeting this criteria, feel free to reach out on Slack for help!

### CODE REVIEW

**Comment by yunnie** [comment]
I couldn't resolve the tests for ```src/views/Tenant/TenantView.test.js```. I think someone more capable than myself will need to take a look at that set. 

I also migrated the tests for ```src/components/Header/header.test.js``` from enzyme to testing-library to be consistent with all the other tests.

File: src/utils/__mocks__/toast.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default jest.fn(() => {})
```
Comment (NickSchimek) [inline src/utils/__mocks__/toast.js]: I'm not seeing this explicitly used anywhere, so I'm wondering how its being used. Is the code that executes when this statement runs in the test? `jest.mock("../../utils/toast")`

Comment (yunnie) [reply src/utils/__mocks__/toast.js]: it's used in ```src/views/Settings/index.test.js``` to check if ```Toast``` was called. Basically it mocks ```Toast``` calls.  I intended to use it to test other ```Toast``` calls in other tests.

Comment (NickSchimek) [reply src/utils/__mocks__/toast.js]: I don't think this file is needed. AFAIK this file sets up a Manual mock that can be used to return data when certain calls are made against it. However, we are only concerned if Toast has been called with certain parameters. So I think converting this to an automock (by deleting this file) should still suit the needs. Let me know if I'm misunderstanding anything as I'm very new to Javascript testing.

Comment (yunnie) [reply src/utils/__mocks__/toast.js]: I'm really new to JS and JS testing too. I'll try the automock and see what happens.

**Review by Chris-Boe — APPROVED** [review]
I pulled this down and rebased it off the most recent development locally and it looks good! Thanks for updating these. I'll try make it more of a priority to get tests written and updated when needed

**Pull request merged**